src/utils/useAuth.tsx

```diff
@@ -0,0 +1,5 @@
+import { getLocalStorage } from "@/utils";
+
+export const useAuth = () => {
```

Comment:
I'm not sure if this is the right name because "use" is used for hooks and technically this function is more like a util instead of a hook because you're not accessing to any state, lifecycle or using other hook. WDYT

Reply:
It's true, it could be `isAuthenticated`